util/unifyfs-shell/src/Makefile.am

```diff
@@ -0,0 +1,26 @@
+bin_PROGRAMS = unifyfs-shell
+
+unifyfs_shell_SOURCES = unifyfs-shell.c \
+                        unifyfs-shell-util.c \
+                        unifyfs-shell-main.c \
+                        unifyfs-shell-status.c \
+                        unifyfs-shell-ls.c \
+                        unifyfs-shell-mkdir.c \
+                        unifyfs-shell-cd.c \
+                        unifyfs-shell-pwd.c \
+                        unifyfs-shell-stat.c
+
+noinst_HEADERS = unifyfs-shell.h
+
+AM_CPPFLAGS = $(MPI_CFLAGS) \
+              -I$(top_srcdir)/common/src \
+              -I$(top_srcdir)/client/src
+
+AM_CFLAGS = -Wall
+
+unifyfs_shell_LDADD = $(MPI_CLDFLAGS) \
+                      $(top_builddir)/common/src/libunifyfs_common.la \
+                      $(top_builddir)/client/src/libunifyfs_gotcha.la -lrt -lm
```

Comment:
what if the configuration didn't include gotcha?

Reply:
I have updated the build script to create two executables (-gotcha and -static)